pkg/api/mlflow/dao/repositories/metric.go

```diff
@@ -219,6 +221,12 @@ func (r MetricRepository) GetMetricHistories(
 		query.Where("metrics.key IN ?", metricKeys)
 	}
 
+	if len(jsonPathValueMap) > 0 {
+		query.Joins("LEFT JOIN contexts on metrics.context_id = contexts.id")
+		sql, args := BuildJsonCondition(query.Dialector.Name(), "contexts.json", jsonPathValueMap)
+		query.Where(sql, args...)
```

Comment:
if you pass `BuildJsonCondition` immediately   to `query.Where`, won't it work?

Reply:
Probably, but then you've got a very busy line of code to unpack...

Reply:
it won't work

Reply:
i think you have to expand the second arg with `...`, which is only possible in this way